subsys/bluetooth/controller/Kconfig

```diff
@@ -345,6 +345,13 @@ config BT_CTLR_LE_POWER_CONTROL
 	  Enable support for LE Power Control feature that defined in the Bluetooth
 	  Core specification, Version 5.3 | Vol 6, Part B, Section 4.6.31.
 
+config BT_CTLR_SDC_CONN_EVENT_TRIGGER
+	bool "Connection Event Trigger"
+	help
+		Enable support for the Connection Event Trigger feature.
+		Connection event triggers are a proprietary SDC feature which can be used
+		to perform tasks in synchronization with radio activity.
+
```

Comment:
```suggestion
config BT_CTLR_SDC_EVENT_TRIGGER
	bool "Event Trigger"
	help
	  Enable support for the Event Trigger feature.
	  The event trigger is  proprietary SDC feature which can be used
	  to perform tasks in synchronization with radio activity.
	  The feature triggers a PPI/DPPI task at the start of every N connection
	  event or start of a scanner/initiator event.
```

Reply:
I will copy some text from sdc_hci_vs.h.
We may extend this to other roles in the future so I prefer to keep the language generic ("start of radio events").